frontend/utils/api.js

```diff
@@ -1,54 +1,71 @@
+import { getAllExtensions } from "showdown";
 import { config } from "../config";
+import axios from "axios";
 
 export function getStrapiURL(path) {
-    return `${
-        process.env.NEXT_PUBLIC_STRAPI_API_URL || "https://bunrieucohuongbeo.longvb.tech"
-    }${path}`;
+  return `${
+    process.env.NEXT_PUBLIC_STRAPI_API_URL ||
+    "https://bunrieucohuongbeo.longvb.tech"
+  }${path}`;
 }
 
 // Helper to make GET requests to Strapi
+// =========new code - 3/11/2021 --- fetch with axios==============
+export const getAboutUs = async (path) => {
```

Comment:
Hàm này không hợp lý
About us là 1 khối của Home page, chúng ta sẽ ko có 1 hàm riêng để lấy dữ liệu của About us mà sẽ có 1 hàm lấy toàn bộ dự liệu của home page, Và chúng ta có một hàm lấy dữ liệu home page ở dưới r